### DISCUSSION TITLE
Welcome to wthrr-the-weathercrab Discussions!

### DISCUSSION THREAD

**Discussion by ttytm** [ttytm/wthrr-the-weathercrab#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with others.
  * Are welcoming and open-minded. Remember open-source is a community that's build together 💪.

**Comment by 0323pin** [comment]
Hi,

Hope you're doing fine. I've just realized it will soon be one year since 1.0.0 release 🎉 

Also, there are 129 commits post 1.1.1 release, so it has been rolling along 😄 
Is a new release due anytime soon?

Regards

**Reply by ttytm** [reply]
Hey @0323pin,

sorry for the delay. It's been some severe health issues with an increased amount of work in the recent weeks. Release is this month 💯 !

**Reply by 0323pin** [reply]
Thank you !
I hope you are feeling well now. Take care of your health.

**Reply by ttytm** [reply]
Thank you @0323pin. Sorry for the delay again. It is big on the list now. Best regards

**Reply by 0323pin** [reply]
@ttytm No rush mate 😃

**Reply by ttytm** [reply]
Finally made it.

To have it on crates.io, `cargo publish` was run from a local machine at the 1.2.0 tags sha due to the old token not being found during the automated CI creates.io publish.

**Reply by 0323pin** [reply]
Thanks for letting me know. Glad to hear it's out 🚀 

I had some issues with my system and I'm almost done with a fresh install. Things take time when you build everything from source 😄 

I'll try to get the package updated by Monday.

**Comment by 0323pin** [comment]
@ttytm [Merged](http://mail-index.netbsd.org/pkgsrc-changes/2024/06/16/msg301948.html). Sorry for the delay.

**Comment by msponer** [comment]
Hey, I just want to say that I am reading this code to learn Rust -- it's excellent: Thank You. Also, your GitHub actions are so well written -- I've also learned quite a lot from how you've structured them.
